test/views/authorised_to_view/AuthorisedToViewSearchSpec.scala

```diff
@@ -173,9 +178,10 @@ class AuthorisedToViewSearchSpec extends SpecBase with MustMatchers {
 
     val sampleView: Document = Jsoup.parse(authorisedToViewInstance.apply(
         form = form,
-        date = date,
-        fileExists = true,
-        isXiEoriEnabled = true)
+        date = Some(date),
+        fileExists = Some(true),
+        isXiEoriEnabled = true,
+        isNotificationPanelEnabled = true)
```

Comment:
Can we have a test that checks there is no panel when `isNotificationPanelEnabled = false`?